### PR TITLE
Initial Commit to Open Source Repo

### PR DESCRIPTION
**Pull request by Alberto-Villarreal**
Signed-off-by: avillarr <alberto.villarreal@intel.com>

### CODE REVIEW

**Comment by Alberto-Villarreal** [comment]
New commit: Added extra include directory to Visual Studio Project to find dpc_common.hpp file:
$(ONEAPI_ROOT)dev-utilities\latest\include</AdditionalIncludeDirectories>

File: DirectProgramming/DPC++/StructuredGrids/iso2dfd_dpcpp/README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+    Computing wavefield in CPU ..
+    Initializing ... 
+    CPU time: 8846 ms
+
+    Final wavefields from device and CPU are equivalent: Success
+    Final wavefields (from device and CPU) written to disk
+    Finished.  
+    [100%] Built target run
+    ```
```
Comment (anjgola) [inline DirectProgramming/DPC++/StructuredGrids/iso2dfd_dpcpp/README.md:140]: The readme needs to have a statement about 'common' file and where to find them.

Comment (Alberto-Villarreal) [reply DirectProgramming/DPC++/StructuredGrids/iso2dfd_dpcpp/README.md:145]: Done. 
Fixed copyright year and added statement about 'common' file to README file
Thanks .

**Comment by Alberto-Villarreal** [comment]
Changes addressed. Ready for approval.

**Pull request merged**